api/api.proto

```diff
@@ -2014,10 +2014,14 @@ message SdkCloudBackupCreateRequest {
   string credential_id = 2;
   // Full indicates if full backup is desired even though incremental is possible
   bool full = 3;
+  // TaskId is the optional unique id of the task performing this backup
```

Comment:
What is the purpose of providing this id? Is it returned in the responses? (it should be documented here) Should the caller provide the id or is the server going to generate one? 

Seems weird to request the id from the caller.

Reply:
This is to allow callers to specify taskID so that the call can idempotent.  If task ID provided already exists , success is returned, if not it will be created and success is returned.